Resources/Textures/Shaders/chameleon.swsl

```diff
@@ -0,0 +1,37 @@
+light_mode unshaded;
+
+uniform sampler2D SCREEN_TEXTURE;
+uniform highp float speed;
```

Comment:
Both the shader & system/component should have a comment mentioning that speed should be a number between -1 and +1.

Might also be worth renaming this to something else because its really more of a cloak/stealth level than a measure of speed.